### ISSUE TITLE
Info representation in doc build is not fully collapsed

### ISSUE THREAD

**Issue by hoechenberger** [mne-tools/mne-python#12130]
### Description of the problem

In the Report, the sections of all Info objects are collapsed, but the vertical space is still occupied

### Steps to reproduce

```Python
# %%
import mne

sample_dir = mne.datasets.sample.data_path()
sample_fname = sample_dir / 'MEG' / 'sample' / 'sample_audvis_raw.fif'

raw = mne.io.read_raw_fif(sample_fname)
raw.crop(tmax=60)

report = mne.Report()
report.add_raw(raw, title="Test")
report.save("/tmp/report.html", overwrite=True)
```

### Expected results

- All sections should be uncollapsed by default
- Collapsing a section should free the corresponding vertical space

### Actual results

![Screenshot 2023-10-24 at 17 07 22](https://github.com/mne-tools/mne-python/assets/2046265/6bc08b2c-f998-475b-8347-c60f8d23b936)

### Additional information

Report has its own collapsible sections

For the purpose of quick data / report assessment, all Info sections should be uncollapsed by default

**Comment by larsoner** [comment]
> For the purpose of quick data / report assessment, all Info sections should be uncollapsed by default

Agreed this should be the case for `mne.Report`. I think for general object `__repr_html__` outside of `mne.Report` we decided to collapse by default in some issue/PR.

**Comment by cbrnr** [comment]
Collapsing in notebooks doesn't make a lot of sense, because e.g. `raw.info` will only show section headers without any information at all.

**Comment by hoechenberger** [comment]
JS inspector in Safari on macOS says:

```
[Error] ReferenceError: Can't find variable: $
	Global Code (30_info.html:1323)
```

The respective line is:
```html
<script type="text/javascript">$(function () { $("time.localized").each(function () { var el = $(this); el.text(new Date(el.attr("datetime")).toLocaleString([], {dateStyle: "medium", timeStyle: "long"})); }); } )</script></p></div>
```

**Comment by larsoner** [comment]
Not related, making new issue

**Comment by larsoner** [comment]
Even after #12145 there is still a bug on Safari on macOS with the collapsing in rendered tutorial HTML, see:

https://github.com/mne-tools/mne-python/pull/12145#issuecomment-1781567722
